.github/workflows/build-push-images-pr.yaml

```diff
@@ -4,15 +4,113 @@ on:
     branches:
       - develop
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - 3.7
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+
+      - name: Test installing development dependencies
+        working-directory: deployment/docker
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r REQUIREMENTS-dev.txt
+
+      - name: Run Flake8 test
+        run: flake8 .
+
+  test:
+    runs-on: ubuntu-latest
+    needs:
+      - lint
+    env:
+      APP_IMAGE: kartoza/projecta-uwsgi
```

Comment:
I'm not sure why there are `core.settings.dev_docker.py` and `core.settings.test_docker.py`. Since we have two settings, the local image build need to have this `APP_IMAGE` parameter, so that you can create docker-compose override that use this parameter.